git-cliff-core/src/changelog.rs

```diff
@@ -907,6 +939,13 @@ mod test {
 		config.git.split_commits = Some(true);
 		config.git.filter_unconventional = Some(false);
 		config.git.protect_breaking_commits = Some(true);
+
+		if let Some(parsers) = config.git.commit_parsers.as_mut() {
+			for parser in parsers.iter_mut().filter(|p| p.footer.is_some()) {
+				parser.skip = Some(true)
+			}
+		}
+
```

Comment:
I'm not sure this is the most maintainable solution to the problem. Let me know if you have a better solution. :relieved:

Reply:
it's good enough!